packages/popover/stories/popover.stories.ts

```diff
@@ -43,7 +43,7 @@ export const Dialog = (): TemplateResult => {
     const placement = radios('Placement', placements, placements.bottom);
     return html`
         <div
-            style="color: var(--spectrum-global-color-gray-800); position: relative"
+            style="color: var(--spectrum-global-color-gray-800); position: relative; display: contents"
```

Comment:
This failure https://9206-158297197-gh.circle-artifacts.com/0/test/visual/screenshots-current/ci/popover--dialog__darkest__large.png points to the possibility of this breaking usage in the wild... it's something we'd actively remove in Crisp in favor of the new containment code, so it shouldn't be a problem there. Does LRWeb have any usage of `sp-overlay` that might break in this way? The issue being the containing `<div>` here had a height of 0 previously due to the `sp-overlay` being `position: absolute` which meant that the math in ```
    max-height: calc(
        100% -
            var(
                --spectrum-dropdown-flyout-menu-offset-y,
                var(--spectrum-global-dimension-size-75)
            )
    );
```
Didn't know what height to make the popover.

Not sure if this is something we should look deeper into as far as the sizing solution above or not. Thoughts?